aliasing/api/context.py

```diff
@@ -204,3 +216,37 @@ def display_name(self):
 
     def __str__(self):
         return f"{self.name}#{self.discriminator}"
+
+
+class AliasCategory:
+    """
+    Represents the category of the Discord channel an alias was invoked in.
+    """
+
+    def __init__(self, category):
+        """
+        :type channel: discord.ChannelCategory
+        """
+        self._name = str(category)
+        self._id = category.id
+
+    @property
+    def name(self):
+        """
+        The name of the category, not including the preceding hash (#).
```

Comment:
what preceding hash?

Reply:
Just something I missed when editing the documentation 
(the functions were initially copied from the channels class)

Fixing in next commit.